antiplag/models.py

```diff
@@ -17,20 +16,18 @@ class SubmissionStatus(models.TextChoices):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
+    def __str__(self):
+        return f"Submission {self.id} ({self.status})"
 
-class Document(models.Model):
-
-    def get_file_path(self, filename):
-        filebase, ext = filename.split(".")
-        filename = f"{filebase}-{str(uuid.uuid4())[:8]}.{ext}"
-        return os.path.join("documents/", filename)
 
+class Document(models.Model):
     class DocumentType(models.TextChoices):
         FILE = "FILE"
         TEXT = "TEXT"
 
-    file = models.FileField(upload_to=get_file_path, null=True)
-    submission = models.ForeignKey(Submission, on_delete=models.CASCADE, null=True)
+    file = models.FileField(upload_to='documents/', null=True)
+    name = models.CharField(max_length=255, null=True)
```

Comment:
It's better to use `blank=True` as empty string instead of null. Check https://docs.djangoproject.com/en/3.1/ref/models/fields/#null for more info.

Reply:
>  If a field has `blank=True`, form validation will allow entry of an empty value.

I'm not getting how you meant that - name is the original file name - "Dokument1.pdf". 
In case of Upload text - neither file nor file name should exist - therefore I thought it should be null, shouldn't it?

Reply:
Sorry, my comment was really confusing.

If string field has `NULL` allowed, then there are 2 ways to represent no string in database - `NULL` or `""`.  Hence `NULL` should not be allowed and only empty string used.

`BLANK` is only used for django form validation and does not matter in this case.

Reply:
Switched to `blank=True`